code/__DEFINES/hud.dm

```diff
@@ -14,3 +14,202 @@
 #define APPEARANCE_UI_IGNORE_ALPHA (RESET_COLOR|RESET_TRANSFORM|NO_CLIENT_COLOR|RESET_ALPHA|PIXEL_SCALE)
 /// Used for HUD objects
 #define APPEARANCE_UI (RESET_COLOR|RESET_TRANSFORM|NO_CLIENT_COLOR|PIXEL_SCALE)
+
+/*
+	These defines specificy screen locations.  For more information, see the byond documentation on the screen_loc var.
+
+	The short version:
+
+	Everything is encoded as strings because apparently that's how Byond rolls.
+
+	"1,1" is the bottom left square of the user's screen.  This aligns perfectly with the turf grid.
+	"1:2,3:4" is the square (1,3) with pixel offsets (+2, +4); slightly right and slightly above the turf grid.
+	Pixel offsets are used so you don't perfectly hide the turf under them, that would be crappy.
+
+	In addition, the keywords NORTH, SOUTH, EAST, WEST and CENTER can be used to represent their respective
+	screen borders. NORTH-1, for example, is the row just below the upper edge. Useful if you want your
+	UI to scale with screen size.
+
+	The size of the user's screen is defined by client.view (indirectly by world.view), in our case "15x15".
+	Therefore, the top right corner (except during admin shenanigans) is at "15,15"
+*/
+
+/proc/ui_hand_position(i) //values based on old hand ui positions (CENTER:-/+16,SOUTH:5)
+	var/x_off = -(!(i % 2))
```

Comment:
bruh

Reply:
```suggestion
	var/x_off = i % 2 ? 0 : -1
```

get outta here with your clever code

Reply:
oh you didnt write this did you, look how silly it is

Reply:
dude that's a heck of a code snippet